src/DynamicData/Cache/ObservableCacheEx.BindSort.cs

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) 2011-2023 Roland Pheasant. All rights reserved.
+// Roland Pheasant licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for full license information.
+
+using System.Collections.ObjectModel;
+using DynamicData.Binding;
+
+namespace DynamicData;
+
+/// <summary>
+/// ObservableCache extensions for BindAndSort.
+/// </summary>
+public static partial class ObservableCacheEx
+{
+    /// <summary>
+    /// Bind sorted data to the specified readonly observable collection.
+    /// </summary>
+    /// <typeparam name="TObject">The type of the object.</typeparam>
+    /// <typeparam name="TKey">The type of the key.</typeparam>
+    /// <param name="source">The source.</param>
+    /// <param name="targetList">The list to bind to.</param>
+    /// <param name="comparer">The comparer to order the resulting dataset.</param>
+    /// <returns>An observable which will emit change sets.</returns>
+    public static IObservable<IChangeSet<TObject, TKey>> BindAndSort<TObject, TKey>(
```

Comment:
Slight nitpick: I think `SortAndBind()` makes more sense, since that's logically what the operator is doing, sorting first, then binding. Put differently, what this operator replaces is `.Sort().Bind()`.

Reply:
Agreed